### ISSUE TITLE
Crash on playback

### ISSUE THREAD

**Issue by MarcSabatella** [musescore/MuseScore#16423]
**Describe the bug**
The attached scores crashes a few measures in during playback.  But, the crash is averted if you change sounds to MS Basic - and it remains good even if you switch back to Muse Sounds, so it seems not totally inherent in using Muse Sounds, but something else about the initial state of the score.

**To Reproduce**
1. Download and unzip the attached file - 
[Sketch #26-Orchestrated - brass.zip](https://github.com/musescore/MuseScore/files/10770075/Sketch.26-Orchestrated.-.brass.zip)
2. Load score
3. Play
4. See error: crash around measure 5

**Expected behavior**
No crash

**Platform information**
 - OS: Linux

**Additional context**
Reproducible both in 4.0.1 and current nightly build of 4.0.2, current Muse Sounds.  But, if you first go to View / Playback setup, activate MS Basic, then immediately reactivate Muse Sounds, then hit OK, and then play, it works fine!

**Comment by MarcSabatella** [comment]
FWIW, I can't reproduce on Windows.  But on Linux, running from the command line, I get an error to the terminal at the point of the crash.  Sometimes it is "malloc(): invalid size (unsorted)", other times it is "double free or corruption (!prev)"

**Comment by cbjeukendrup** [comment]
@MarcSabatella could you please check if this is still actual?

**Comment by MarcSabatella** [comment]
It no longer crashes.  However, I note the trumpet defaults to MS Basic, not sure if that was the case in 4.0.2 as I can't run that anymore (something about wl_proxy_flags; there is probably a workaround but I don't know what it is).

**Comment by cbjeukendrup** [comment]
When I open that score with 4.0.2 on my Mac, it also defaults to MS Basic for the trumpets. Either this happened back then too, or it is influenced by the latest MuseSampler. 

So I don't really know what to do with this issue now. But I'm inclined to close it, because it isn't really reproducible anymore and most likely it was just another instance of that same very popular MuseSounds-on-Linux crash, which seems to have been fixed.

**Comment by MarcSabatella** [comment]
Makes sense to me.  Wasn’t my score anyhow; I was just reporting on behalf of someone else :-)
